README.md

```diff
@@ -24,16 +24,7 @@ We are in the process of adding Python 3 support.
 
 1. Create or activate a virtualenv for working on govdelivery.
 1. `pip install -e .`
```

Comment:
If you want to install the tests using the `testing` extras, you need to to `pip install -e '.[testing]'. Otherwise they'll fail because you're missing responses.

Reply:
D'oh, yes. I discovered this also, but didn't update the instructions accordingly. Thanks!

Reply:
Fixed in 88ee0a6